pkg/orchestrator/provider/common/utils.go

```diff
@@ -0,0 +1,89 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/sirupsen/logrus"
+)
+
+// LogarithmicFormula represents the formula y = a * ln(x) + b.
+type LogarithmicFormula struct {
+	a float64
+	b float64
+}
+
+// Evaluate receive an x value and returns the y value of the formula.
+func (lf *LogarithmicFormula) Evaluate(x float64) float64 {
+	return lf.a*math.Log(x) + lf.b
```

Comment:
pretty surprised this isn't a linter error.
```suggestion
	return lf.a * math.Log(x) + lf.b
```